query/src/main/scala/filodb/query/exec/rangefn/AggrOverTimeFunctions.scala

```diff
@@ -808,6 +897,23 @@ class QuantileOverTimeChunkedFunctionL(funcParams: Seq[FuncArgs])
   }
 }
 
+class MedianAbsoluteDeviationOverTimeChunkedFunctionL
+  extends MedianAbsoluteDeviationOverTimeChunkedFunction with ChunkedLongRangeFunction {
+  final def addTimeLongChunks(longVectAcc: MemoryReader,
+                              longVect: BinaryVector.BinaryVectorPtr,
+                              longReader: bv.LongVectorDataReader,
+                              startRowNum: Int,
+                              endRowNum: Int): Unit = {
+
+      var rowNum = startRowNum
+      val it = longReader.iterate(longVectAcc, longVect, startRowNum)
+      for (rowNum <- startRowNum to endRowNum) {
```

Comment:
Since you don't use the `rowNum` variable, change it to `_`, also i assume `next` doesn't throw any error as we dont have check for `hasNext` before calling `next`.